core/interfaces/i_registrable_field.js

```diff
@@ -11,21 +11,25 @@
 
 'use strict';
 
-goog.provide('Blockly.IRegistrableField');
+goog.module('Blockly.IRegistrableField');
+goog.module.declareLegacyNamespace();
 
 goog.requireType('Blockly.Field');
 
+
 /**
  * A registrable field.
  * Note: We are not using an interface here as we are interested in defining the
  * static methods of a field rather than the instance methods.
  * @typedef {{
- *     fromJson:Blockly.IRegistrableField.fromJson
+ *     fromJson:IRegistrableField.fromJson
  * }}
  */
-Blockly.IRegistrableField;
+let IRegistrableField;
 
 /**
  * @typedef {function(!Object): Blockly.Field}
  */
-Blockly.IRegistrableField.fromJson;
+IRegistrableField.fromJson;
+
+exports = IRegistrableField;
```

Comment:
I see in #5060 and #5079 @gonfunko has omitted the blank line between the definition and the export.  I like that because it makes clear the export is directly related to the definition above.

Reply:
Those files differ in that they aren't exporting a single class.

I agree that it makes sense to put the export directly below the function definition, but not in the case of a class.
I think that the newline here makes it easier to read and removing it would make it seem like it is directly related to the function definition for `IRegistrableField.fromJson;`

Reply:
Yes, I see that now, and I agree with you.